tests/tests.py

```diff
@@ -82,12 +82,15 @@ def run(
     check_md5=True,
     cores=3,
     set_pythonpath=True,
+    cleanup=True,
     **params
 ):
     """
     Test the Snakefile in path.
     There must be a Snakefile in the path and a subdirectory named
-    expected-results.
+    expected-results. If cleanup is False, we return the temporary
+    directory to the calling test for inspection, and the test should
+    clean it up.
```

Comment:
Very nice idea!

Reply:
Yes this will be very helpful for me to (slowly) figure out how things work!